client/pk-monitor.c

```diff
@@ -128,6 +128,42 @@ pk_monitor_adopt_cb (PkClient *_client, GAsyncResult *res, gpointer user_data)
 	}
 }
 
+static gchar*
+pk_monitor_get_caller_info (GDBusProxy *bus_proxy, const gchar *name)
```

Comment:
Might be better to call the argument `bus_name` rather than just `name` to make it a bit clearer it’s a unique or well-known D-Bus name.